torch/distributed/fsdp/_state_dict_utils.py

```diff
@@ -634,6 +638,10 @@ def _sharded_pre_load_state_dict_hook(
                 tensor, local_tensor, group=fsdp_state.process_group
             )
         tensor = tensor.narrow(0, 0, param_numel).reshape(param.size())
+        if fqn_to_param_ext.get(fqn) is not None:
+            ext = fqn_to_param_ext.get(fqn)
```

Comment:
I wonder if we are allowed to use the walrus operator now.

Reply:
Good question, we should since it's on python 3.8